docs/guide/plugins.md

```diff
@@ -65,23 +67,28 @@ end
 
 -- Callback if the menu item is executed
 function run()
--- ADD CODE
+-- ADD CODE LIKE
+-- app.uiAction({["action"]="ACTION_TOOL_PEN"})
+-- TO SWITCH TO THE PEN TOOL, OR
+-- app.changeToolColor({["color"] = 0xff0000, ["selection"] = true})
+-- TO CHANGE THE COLOR OF THE CURRENT TOOL TO RED
 end
 
-``` 
+```
 
 ## Plugin API 
 
-The Lua Plugin can execute a (small) number of Xournal++ functions to interact with Xournal++. Those are defined in the 
-[Plugin API](https://github.com/xournalpp/xournalpp/blob/master/src/plugin/luapi_application.h#L237-L242?)
+The Lua Plugin can execute a (small) number of Xournal++ functions to interact with Xournal++. Those are defined and documented in the 
+[Plugin API](https://github.com/xournalpp/xournalpp/blob/master/src/plugin/luapi_application.h#311-317?).
```

Comment:
This should be a permalink, otherwise this link will get outdated very frequently.

Reply:
Fixed